### PR TITLE
fix(cactus-example-electricity-trade): enable tsconfig strict flag an…

### PR DESCRIPTION
**Pull request by rwAT17**
…d fix all the warnings

Fixed all warnings when strict flag is enabled

Closes: #2144

Signed-off-by: Tomasz Awramski <tomasz.awramski@fujitsu.com>

### CODE REVIEW

File: examples/cactus-example-electricity-trade/BusinessLogicElectricityTrade.ts [hunk 1]
```diff
@@ -20,10 +20,10 @@ import {
 } from "@hyperledger/cactus-cmd-socketio-server";
 import { makeRawTransaction } from "./TransactionEthereum";
 
-const fs = require("fs");
-const yaml = require("js-yaml");
+// const fs = require("fs");
+// const yaml = require("js-yaml");
```
Comment (izuru0) [inline examples/cactus-example-electricity-trade/BusinessLogicElectricityTrade.ts:24]: probably you can simply remove these lines?

Comment (rwAT17) [reply examples/cactus-example-electricity-trade/BusinessLogicElectricityTrade.ts]: fixed

**Comment by takeutak** [comment]
@rwAT17 Thanks for contribution and I'm very sorry to wait you for a while!  I'm asking @izuru0 to review this PR again.  He will review it again next Monday, so please wait for a minute?

**Review by izuru0 — APPROVED** [review]
LGTM

**Review by petermetz — APPROVED** [review]
@rwAT17 Thank you, LGTM!

**Pull request merged**